lib/types/application/ApplicationBuilder.js

```diff
@@ -82,16 +84,23 @@ class ApplicationBuilder extends AbstractBuilder {
 			});
 		});
 
-		this.addTask("generateAppPreload", () => {
+		var aComponentPreload = project.builder.componentPreload;
```

Comment:
No hungarian notation to be consistent with the general style applied here.

Reply:
done